.github/pull_request_assignment.yml

```diff
@@ -9,7 +9,7 @@
 - rule:
     # OpenAI data-plane PR
     paths:
-      - "specification/cognitiveservices/data-plane/AzureOpenAI/**"
+      - "specification/cognitiveservices/data-plane/OpenAI.Inference/**"
```

Comment:
We likely need to clean up our paths a bit, but these are the two that I think are of interest:

- specification/cognitiveservices/data-plane/AzureOpenAI
- specification/cognitiveservices/OpenAI.Inference

Compiling in the second one outputs to the first one.

The old path "looks correct" for the JSON; should this just add the second one to check changes without JSON (ignoring that they probably *should*)?